pkg/crawl/identify.go

```diff
@@ -306,6 +266,9 @@ func (i *Log4jIdentifier) lookForClassFileMatch(path, filename string, size int6
 		}
 		if hashMatch {
 			i.printDetailedHashFinding(path, finding)
+			if obfuscated {
```

Comment:
Let's call `i.classMeetsObfuscationThreshold(path, filename)` here and so check for obfuscation _only_ if the hash matches.

Reply:
We need to check if the file is obfuscated in order to decide to attempt a hash match though - we can't wait until this point to compute this.

Reply:
Ah, you are right, yeah. Sorry, can't believe I missed that.